### ISSUE TITLE
Battery segment showing as charging color/icon when it should show as discharging 

### ISSUE THREAD

**Issue by devhawk** [JanDeDobbeleer/oh-my-posh#644]
### Prerequisites

- [X] I have read and understand the `CONTRIBUTING` guide
- [X] I looked for duplicate issues before submitting this one

### Description

Battery segment shows as charging when power is disconnected.

Problem seems to be isolated to Windows. I have configured my WSL distros on this box to use the oh-my-posh-wsl binary and the same theme as I do on Windows (bash instead of powershell tho). On Ubuntu, the battery segment displays correctly when discharging.

### Environment

- Oh my Posh version: 3.137.0
- Theme: custom
- Operating System: Windows 10 20H2
- Shell: Powershell
- Terminal: Windows Terminal

### Steps to Reproduce

Battery segment of my theme:

``` json
                {
                    "type": "battery",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B0",
                    "foreground": "#000000",
                    "background": "#ffeb3b",
                    "properties": {
                        "battery_icon": "",
                        "discharging_icon": "\uE234 ",
                        "charging_icon": "\uF1E6 ",
                        "charged_icon": "\uF1E6 ",
                        "color_background": true,
                        "charged_color": "#4caf50",
                        "charging_color": "#40c4ff",
                        "discharging_color": "#ff5722",
                        "postfix": "\uF295 ",
                        "display_charging": true
                    }
                },
```

**Expected behavior:** [What you expected to happen]

How it looks on WSL:
![image](https://user-images.githubusercontent.com/8965/114641842-ab191d80-9c87-11eb-80aa-75b6bfff5658.png)

**Actual behavior:** [What actually happened]

How it looks on Windows:
![image](https://user-images.githubusercontent.com/8965/114641609-4a89e080-9c87-11eb-8a12-b200faea0fba.png)

**Comment by lnu** [comment]
Are you sure you're using the same theme? 
I tried on my side and I always get the same layout if I use the same theme on windows and wsl.
If I try your config, I  always get the second one:
![image](https://user-images.githubusercontent.com/1829553/114659954-c3735180-9cf4-11eb-8223-46668097b396.png)

And is it always different because I noticed that sometimes if I open two different terminal windows I don't have the same color/icon but if I clear both then it's ok. The reading can be different.

And there's something different in the readings between wsl and windows(not omp but distatus/battery which is used to get battery data):
![image](https://user-images.githubusercontent.com/1829553/114661092-b0fa1780-9cf6-11eb-80e8-a338cb11d8a5.png)

**Comment by JanDeDobbeleer** [comment]
The code is actually the same, so there's no way oh-my-posh can inform a difference in behaviour unless the OS (or distaffs/battery) reports something else. The only thing impacting this can be when Windows reports maybe more batteries than WSL, which indeed maps the status the most logical way.

**Comment by devhawk** [comment]
> Are you sure you're using the same theme?

Yes, I'm sure

> The code is actually the same, so there's no way oh-my-posh can inform a difference in behaviour unless the OS (or distaffs/battery) reports something else. The only thing impacting this can be when Windows reports maybe more batteries than WSL, which indeed maps the status the most logical way.

This is a surface book which has multiple batteries, so that must be the source of the behavior difference. I charged overnight and unplugged, now my battery toolbar widget lists both batteries as "100% and in use"

![image](https://user-images.githubusercontent.com/8965/114742361-83b26700-9d00-11eb-941a-057cbde3c1b9.png)

I'm not sure why OMP on Windows displays this state using the charging icon + color

![image](https://user-images.githubusercontent.com/8965/114742537-b52b3280-9d00-11eb-99e9-a952998e5c8b.png)

**Comment by JanDeDobbeleer** [comment]
@devhawk would it make more sense for you to see both batteries btw? I'll validate the logic as maybe I made a silly mistake.

**Comment by devhawk** [comment]
> @devhawk would it make more sense for you to see both batteries btw? I'll validate the logic as maybe I made a silly mistake.

I would think displaying a single combined battery segment makes most sense. Thanks for taking a look at the logic. Let me know how I can help. Unfortunately, my go skills are VERY limited so I can't really help troubleshoot locally

**Comment by JanDeDobbeleer** [comment]
@devhawk I tweaked the mapping and adjusted it so it can be tested. Can you validate the latest release?

**Comment by blauwkeut** [comment]
Also had the same behaviour on my Windows machine, but seems resolved by the v3.137.3 release...

**Comment by devhawk** [comment]
> @devhawk I tweaked the mapping and adjusted it so it can be tested. Can you validate the latest release?

v3.137.3 also appears to have resolved this issue on my Surface Book. Thanks @JanDeDobbeleer!
